src/main/kotlin/service/PersonalAccountManagementService.kt

```diff
@@ -0,0 +1,23 @@
+package service
+
+import exchange.Exchange
+import transaction.Transaction
+import user.User
+import wallet.Wallet
+import java.time.LocalDateTime
+
+interface PersonalAccountManagementService {
```

Comment:
I understand that we all are were taught to create interfaces even when dealing with Spring, but in this case these interfaces for services are redundant. Try not to overload packages with interfaces which will be implemented only once